tf-modules/cumulus-rds-tf/main.tf

```diff
@@ -94,11 +95,11 @@ resource "aws_rds_cluster" "cumulus" {
   preferred_backup_window = var.backup_window
   db_subnet_group_name    = aws_db_subnet_group.default.id
   apply_immediately       = var.apply_immediately
+  storage_encrypted       = true
 
-  scaling_configuration {
+  serverlessv2_scaling_configuration {
```

Comment:
Presumably this naming change was required for V2?

Reply:
Yes, this property name is different for v2. Ref: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster#rds-serverless-v2-cluster